tests/staticTree.js

```diff
@@ -13,6 +13,7 @@ suite['should convert actions to objects'] = function (test) {
     duration: 0,
     path: [0],
     mutations: [],
+    services: [],
```

Comment:
Hm, yeah, I agree. `serviceCalls` is good! Should I make another commit or do we change existing? I can try to change existing myself :)